src/widgetastic_patternfly4/card.py

```diff
@@ -0,0 +1,59 @@
+from widgetastic.log import call_sig
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import ClickableMixin
+from widgetastic.widget import ParametrizedView, ParametrizedViewRequest, GenericLocatorWidget
+from widgetastic.widget import Widget, View
+from widgetastic.xpath import quote
+from itertools import count
+
+class Card(GenericLocatorWidget):
+
+    def __init__(self, parent, locator=None, logger=None, **kwargs):
+        View.__init__(self, parent, logger=logger, **kwargs)
+        self.locator = locator or ".//article[contains(@class, 'pf-c-card')]"
```

Comment:
Can we have OUIA locator here:

```python
self.locator = locator or ".//*[@data-component-type='PF4/Card']"
```

Reply:
structurally that would be incorrect as ouia is not generally implemented, but we can go with it